requirements.txt

```diff
@@ -2,4 +2,6 @@ PyYAML==6.0.1
 pandas==2.0.3
 psycopg[binary]==3.1.9
 pyserial==3.5
-schema==0.7.5
\ No newline at end of file
+schema==0.7.5
+psutil==5.9.5
+xgboost==1.7.6
```

Comment:
why is this requirement here? XGBoost should be only required if you install the reporter .

The XGBoost reporter needs in any case way more dependencies to work ...?

Reply:
I wanted to benchmark against the xgboost reporter and it didn't work and there was no reference in the documentation [1]. Now looking into this we do the git submodule update in the install script but then don't install the packages as described in [2]. I think we should make this consistent. Either do the submodule init and install the pip packages and give the user a working system or don't init the submodules in the install. 

[1] https://docs.green-coding.berlin/docs/measuring/metric-providers/psu-energy-xgboost-system/
[2] https://docs.green-coding.berlin/docs/installation/installation-linux/#xgboost